drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -9,6 +9,7 @@
  * FMAC IF Layer of the Wi-Fi driver.
  */
 
+#ifndef CONFIG_NRF700X_RADIO_TEST
```

Comment:
This seems unnecessary if you're excluding in `CMakeLists.txt`?

Reply:
Removed the same.